musicplayer/mpdproxy.py

```diff
@@ -0,0 +1,81 @@
+# pylint: disable=no-member
+"""mpdfunk provides basic connectivity and song information from a locally running MPD instance"""
+import logging
+from mpd import MPDClient, MPDError
+
+# TODO(camcast): Need to run this code with close to prod environment to fully valdiate
```

Comment:
Need to remove since I have already gone through and validate this new class structure.

Reply:
Done